src/gui/rss/feedlistwidget.cpp

```diff
@@ -235,16 +235,16 @@ QTreeWidgetItem *FeedListWidget::createItem(RSS::Item *rssItem, QTreeWidgetItem
     QIcon icon;
     if (auto feed = qobject_cast<RSS::Feed *>(rssItem)) {
         if (feed->isLoading())
-            icon = QIcon(QStringLiteral(":/icons/loading.png"));
+            icon = UIThemeManager::instance()->getIcon("RSS.LoadingFeed");
         else if (feed->hasError())
-            icon = UIThemeManager::instance()->getIcon(QStringLiteral("unavailable"));
+            icon = UIThemeManager::instance()->getIcon(QStringLiteral("RSS.UnAvailable"));
         else if (!feed->iconPath().isEmpty())
             icon = QIcon(feed->iconPath());
         else
-            icon = UIThemeManager::instance()->getIcon(QStringLiteral("application-rss+xml"));
+            icon = UIThemeManager::instance()->getIcon(QStringLiteral("RSS.Available"));
     }
     else {
-        icon = UIThemeManager::instance()->getIcon("inode-directory");
+        icon = UIThemeManager::instance()->getIcon("RSS");
```

Comment:
Where it is actually used? I mean I can't able to understand in which context this icon is actually used thats why such a weird name.